src/borg/helpers/parseformat.py

```diff
@@ -405,6 +414,16 @@ def normpath_special(p):
             p = os.path.normpath(p)
             return ('/.' + p) if relative else p
 
+        m = self.win_file_re.match(text)
```

Comment:
like it is, this will allow drive letters on linux, bsd and mac.

Reply:
Fixed